deployment/pypi/install.ps1

```diff
@@ -46,7 +46,7 @@ yarn
 yarn build
 Copy-Item config -Destination .\dist\ -Recurse -Force
 cd $CWD\..\..\src\webui
-yarn
+yarn --ignore-engines
```

Comment:
Why this?

Reply:
to fix error: `error jest-diff@26.4.2：The engine "node" is incompatible with this module. Expected version ">= 10.14.2". Got "10.13.0"`

Reply:
So why not upgrade node? I'm have been using 10.22.0 in my dev environment and it works fine.

Reply:
BTW I'm looking forward to bump node to v14 or v12 in nni 2.0.
Node v10 is running out of its life. There's only half a year left.

Reply:
Agreed. Let's bump into at least 10.14.2. 10.22.0 sounds charming to me. 👍